### ISSUE TITLE
configuration not working

### ISSUE THREAD

**Issue by bakree786** [who1s/modlishka-config#2]
when inserting password it says Can't sign in to gmail (This browser or app may not be secure)

**Comment by wizzard-lizzard** [comment]
Yeah, looks like Google might detect this type of activity now. There may be a way around that, but some research will be needed.
